synapse/tests/test_lib_storm.py

```diff
@@ -4475,6 +4473,25 @@ async def test_storm_copyto(self):
             self.eq(('inet:ipv4', 0x05050505), nodes[0].ndef)
             self.eq('bar', await core.callStorm('media:news return($node.data.get(foo))', opts=opts))
 
+            oldn = await core.nodes('[ inet:ipv4=2.2.2.2 ]', opts=opts)
+            newn = await core.nodes('[ inet:ipv4=2.2.2.2 ]')
+            self.ne(oldn[0].props['.created'], newn[0].props['.created'])
```

Comment:
Do we need a quick sleep here to ensure we don't "win" the race of matching time stamps?